_pages/sharedtask/2024.md

```diff
@@ -0,0 +1,149 @@
+
+| Q   | A 65-year-old woman comes to the physician for a follow-up examination after blood pressure measurements were 175/105 mm Hg and 185/110 mm Hg 1 and 3 weeks ago, respectively. She has well-controlled type 2 diabetes mellitus. Her blood pressure now is 175/110 mm Hg. Physical examination shows no other abnormalities. Antihypertensive therapy is started, but her blood pressure remains elevated at her next visit 3 weeks later. Laboratory studies show increased plasma renin activity; the erythrocyte sedimentation rate and serum electrolytes are within the reference ranges. Angiography shows a high-grade stenosis of the proximal right renal artery; the left renal artery appears normal. Which of the following is the most likely diagnosis?  |
+|-----|:--|
+| (A) | Atherosclerosis  |
+| (B) | Congenital renal artery hypoplasia |
+| (C) | Fibromuscular dysplasia  |
+| (D) | Takayasu arteritis  |
+| (E) | Temporal arteritis  |
+
+*Table 1: An example of a practice item from the USMLE Step 1 Sample Test Questions at usmle.org*
+
+The part describing the case is referred to as stem, the correct answer is referred to as key, and the incorrect answer options are known as distractors. All items are MCQs that test medical knowledge and were written by experienced subject matter experts following a set of guidelines, stipulating adherence to a standard structure. These guidelines require avoidance of “window dressing” (extraneous material not needed to answer the item), “red herrings” (information designed to mislead the test-taker), and grammatical cues (e.g., correct answers that are longer or more specific than the other options). The goal of standardizing items in this manner is to produce items that vary in their difficulty and discriminating power due only to differences in the medical content they assess. 
+
+The items were administered within a standard nine-hour exam. For this shared task, the item characteristic data was derived from first-time examinees from accredited US and Canadian medical schools.
+
+Each item is tagged with the following item characteristics:
+
+- **Item difficulty**: A measure of item difficulty where higher values indicate a higher proportion of examinees answered the item correctly (less difficult items).
+
+- **Time intensity**: arithmetic mean response time, measured in seconds, across all examinees who attempted a given item in a live exam. This includes all time spent on the item from the moment it is presented on the screen until the examinee moves to the next item, as well as any revisits.
+
+
+The data is structured as follows:
+
+| ItemNum | ItemStem_Text | Answer__A | Answer__B| Answer__C| Answer__D| Answer__E| (…) | Answer_Text | PVALUE | Response_Time | EXAM |
+|---|---|---|---|---|---|---|---|---|---|---|---|
```

Comment:
There is a problem with this table format. The table is not correctly visualized.

Reply:
Should be resolved now :)